diff --git a/Makefile b/Makefile
index 700313b..5876048 100644
--- a/Makefile
+++ b/Makefile
@@ -309,17 +309,8 @@ deb:
 # Bless a new neubot release (sources and Debian).
 #
 release:
-	if ! [ "$(VERSION)" = "$$(git describe --tags)" ]; then		\
-	    echo "error: not at a release point" 2>&1;			\
-	    echo "Makefile version is: $(VERSION)" 2>&1;		\
-	    echo "git describe is: $$(git describe --tags)" 2>&1;	\
-	    exit 1;							\
-	fi
 	make clean
 	make deb
-	make archive
-	./M-Lab/deploy.sh -n
-	./scripts/sign_all
 	./scripts/update_apt
 	cd dist && find -type f -exec chmod 644 {} \;
 	cd dist && find -type d -exec chmod 755 {} \;
