diff --git a/Debian/control/control b/Debian/control/control
index 021ff8b..85e3beb 100644
--- a/Debian/control/control
+++ b/Debian/control/control
@@ -1,5 +1,5 @@
 Package: neubot
-Version: 0.4.15.5-1
+Version: 0.4.15.5-2
 Architecture: all
 Maintainer: Simone Basso <bassosimone@gmail.com>
 Installed-Size: @SIZE@
diff --git a/Debian/control/control-nox b/Debian/control/control-nox
index 84472b1..a4d8dbc 100644
--- a/Debian/control/control-nox
+++ b/Debian/control/control-nox
@@ -1,5 +1,5 @@
 Package: neubot-nox
-Version: 0.4.15.5-1
+Version: 0.4.15.5-2
 Architecture: all
 Maintainer: Simone Basso <bassosimone@gmail.com>
 Installed-Size: @SIZE@
diff --git a/Makefile b/Makefile
index 51c8367..9a7a33b 100644
--- a/Makefile
+++ b/Makefile
@@ -232,8 +232,8 @@ install:
 # Make package for Debian/Ubuntu/Mint
 #
 
-DEB_PACKAGE = dist/neubot-$(VERSION)-1_all.deb
-DEB_PACKAGE_NOX = dist/neubot-nox-$(VERSION)-1_all.deb
+DEB_PACKAGE = dist/neubot-$(VERSION)-2_all.deb
+DEB_PACKAGE_NOX = dist/neubot-nox-$(VERSION)-2_all.deb
 
 _deb_data:
 	make -f Makefile _install DESTDIR=dist/data PREFIX=/usr \
